spec.bs

```diff
@@ -286,6 +286,7 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
   1. [=Queue a global task=] on [=DOM manipulation task source=], given |global|, to resolve |p|
     with |winner|'s [=generated bid/render=]. TODO: resolve |p| with urn-uuid, instead of a
     URL.
+  1. Run [=generate reporting URLs=] with |auctionConfig| and |winner|.

```

Comment:
do we need a render ads step before this, since we do report after the winning ad is navigated to, right?

Reply:
I think this confirms what I thought: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/ad_auction_service_impl.cc;drc=48340c1e35efad5fb0253025dcc36b3a9573e258;bpv=1;bpt=1;l=604